libvmaf/src/feature/feature_extractor.c

```diff
@@ -287,7 +287,7 @@ get_fex_list_entry(VmafFeatureExtractorContextPool *pool,
     entry.ctx_list = malloc(ctx_array_sz);
     if (!entry.ctx_list) goto fail;
     memset(entry.ctx_list, 0, ctx_array_sz);
-    int err = vmaf_dictionary_copy(&opts_dict, &entry.opts_dict);
+    vmaf_dictionary_copy(&opts_dict, &entry.opts_dict);
```

Comment:
@kylophone These changes should be mostly safe, this is one that I wasn't sure about, currently `err` is being thrown away but if I add `if (err) goto fail;` the unit tests fail because they pass a null dict to `vmaf_fex_ctx_pool_aquire`. So it's not clear to me if we want to fail on this.